### PR TITLE
openstack: complete move of OpenStack under the downstream machine api

### PR DESCRIPTION
**Pull request by flaper87**

### CODE REVIEW

**Comment by flaper87** [comment]
/hold
cc @tomassedovic @trown

**Comment by tomassedovic** [comment]
@flaper87 just checking: this depends on https://github.com/openshift/cluster-api-provider-openstack/pull/16 right?

File: pkg/asset/machines/openstack/machines.go [hunk 1]
```diff
@@ -43,14 +43,14 @@ func Machines(clusterID string, config *types.InstallConfig, pool *types.Machine
...
 	for idx := int64(0); idx < total; idx++ {
 		az := ""
 		provider, err := provider(clusterID, clustername, platform, mpool, osImage, az, role, userDataSecret)
 		if err != nil {
 			return nil, errors.Wrap(err, "failed to create provider")
 		}
-		machine := clusterapi.Machine{
+		machine := machineapi.Machine{
 			TypeMeta: metav1.TypeMeta{
 				APIVersion: "cluster.k8s.io/v1alpha1",
```
Comment (enxebre) [inline pkg/asset/machines/openstack/machines.go]: you'll need to update the APIVersion here

Comment (enxebre) [reply pkg/asset/machines/openstack/machines.go]: and namespace to `openshift-machine-api`

Comment (flaper87) [reply pkg/asset/machines/openstack/machines.go]: good catch, thanks

Comment (flaper87) [reply pkg/asset/machines/openstack/machines.go]: There's a separate patch for the `openshift-machine-api` change

File: pkg/asset/manifests/cluster_k8s_io.go [hunk 2]
```diff
@@ -53,7 +53,7 @@ func (c *ClusterK8sIO) Generate(dependencies asset.Parents) error {
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      installconfig.Config.ObjectMeta.Name,
-			Namespace: "openshift-cluster-api",
```
Comment (enxebre) [inline pkg/asset/manifests/cluster_k8s_io.go]: If you are using this object on the actuator, you probably need to update the APIVersion version here `APIVersion: "cluster.k8s.io/v1alpha1"` to be consistent with what the actuator is taking

Comment (flaper87) [reply pkg/asset/manifests/cluster_k8s_io.go]: should be fine. I can prolly change this in a follow-up patch

**Pull request merged**